packages/twenty-front/src/modules/object-record/record-table/record-table-cell/hooks/useUpsertRecord.ts

```diff
@@ -51,14 +57,14 @@ export const useUpsertRecord = ({
         if (isDefined(recordTablePendingRecordId) && isDefined(draftValue)) {
           createOneRecord({
             id: recordTablePendingRecordId,
-            name: draftValue,
```

Comment:
how was it working before? :p

Reply:
Haha probably not working for custom objects with a different label identifier. But I'm not sure many people went as far as customizing the label identifier on their custom objects